projects/robots/festo/robotino3/protos/Robotino3Platform.proto

```diff
@@ -11,16 +11,21 @@ PROTO Robotino3Platform [
   field  SFString    name         "Platform for Robotino 3"      # Is `Solid.name`.
   field  SFString    model        "Festo - Robotino's platform"  # Is `Solid.model`.
   field  MFNode      extension    []                             # Extends the platform.
+  field  SFFloat     height       0.530447                       # Height of the platform.
 ]
 {
+  %{
+    HEIGHT = fields.height.value
```

Comment:
```suggestion
    local height = fields.height.value
```

Reply:
Here doesn't need to be global, and typically variable names are lowercase according to codestyle

Reply:
Thanks for the review!